src/main/java/com/google/sps/servlets/ChatServlet.java

```diff
@@ -49,8 +49,9 @@
      */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        ServletUtils.enforceUserLogin(request, response);
-        Entity userEntity = ServletUtils.getUserEntity(request.getUserPrincipal().getName());
+        // ServletUtils.enforceUserLogin(request, response);
+        // Entity userEntity = ServletUtils.getUserEntity(request.getUserPrincipal().getName());
+        Entity userEntity = ServletUtils.getUserEntity("example@test.com");
```

Comment:
Is this hard coded line going to be changed to the commented out lines at some point? If so make a TODO to clarify